src/components/axes/timeAxis.ts

```diff
@@ -240,8 +242,14 @@ export module Axis {
 
     public _computeHeight() {
       var textHeight = this._measurer.measure().height;
-      this._tierHeights = this._tierLabelPositions.map((pos: string) =>
-        textHeight + this.tickLabelPadding() + ((pos === "between") ? 0 : this._maxLabelTickLength()));
+      var maximumTiers = _Util.Methods.max(this._possibleTimeAxisConfigurations.map((config: TimeAxisConfiguration) => config.length), 0);
```

Comment:
Is it possible to actually know the number of tiers that we will have instead of the maximum?  If maximum > actual, then we will be left with extra space right?